src/Commands/DownloadCommand.php

```diff
@@ -18,6 +18,8 @@ class DownloadCommand extends Command
 
     public function handle()
     {
+        ini_set('memory_limit', '1024M');
```

Comment:
you can also stream the downloaded file like [this](https://github.com/tonysm/tailwindcss-laravel/commit/00e0ef39df2ee4a95b65770c54d2471059283d6b), then you don't have to bump the memory limit.

Reply:
Makes sense, changed. Thanks!